tests/test_sageintacct/test_utils.py

```
@@ -1497,3 +1497,39 @@ def test_construct_journal_entry_with_single_credit_line(create_journal_entry, d
+    invalidate_sage_intacct_credentials(workspace_id)
+    assert not mocked_patch.called
+
+    # Should call patch_integration_settings with the correct arguments if sage_intacct_credentials.is_expired is False
+    sage_intacct_credentials.is_expired = False
+    sage_intacct_credentials.save()
+
+    invalidate_sage_intacct_credentials(workspace_id)
+
+    args, kwargs = mocked_patch.call_args
+    assert args[0] == workspace_id
+    assert kwargs['is_token_expired'] == True
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if kwargs['is_token_expired']:` for truth checks

Replace with `kwargs['is_token_expired']`

(E712)